packages/azure_functions/data_stream/functionapplogs/manifest.yml

```
@@ -0,0 +1,59 @@
+title: Collect Azure Functions logs
+type: logs
+streams:
+  - input: "azure-eventhub"
+    enabled: false
+    template_path: "azure-eventhub.yml.hbs"
+    title: Azure Functions logs
+    description: Collect Azure Functions logs using azure-eventhub input
+    vars:
+      - name: preserve_original_event
+        required: true
+        show_user: true
+        title: Preserve original event
+        description: Preserves a raw copy of the original event, added to the field `event.original`
+        type: bool
+        multi: false
+        default: false
+      - name: storage_account_container
+        type: text
+        title: Storage Account Container
+        multi: false
+        required: false
+        show_user: false
+        description: >
+          The storage account container where the integration stores the checkpoint data for the consumer group. It is an advanced option to use with extreme care. You MUST use a dedicated storage account container for each Azure log type (activity, sign-in, audit logs, and others). DO NOT REUSE the same container name for more than one Azure log type. See [Container Names](https://docs.microsoft.com/en-us/rest/api/storageservices/naming-and-referencing-containers--blobs--and-metadata#container-names) for details on naming rules from Microsoft. The integration generates a default container name if not specified.
+
+      - name: tags
+        type: text
+        title: Tags
+        multi: true
+        required: true
+        show_user: false
+        default:
+          - azure-functions
+          - forwarded
+      - name: processors
+        type: yaml
+        title: Processors
+        multi: false
+        required: false
+        show_user: false
+        description: >-
+          Processors are used to reduce the number of fields in the exported event or to enhance the event with metadata. This executes in the agent before the logs are parsed. See [Processors](https://www.elastic.co/guide/en/beats/filebeat/current/filtering-and-enhancing-data.html) for details.
+      - name: sanitize_newlines
+        title: Sanitize New Lines
+        description: Sanitizes New Lines
```

Align the description with #6357 

```suggestion
        description: Removes new lines in logs to ensure proper formatting of JSON data and avoid parsing issues during processing.
```